scripts/tools/release-cli/release-cli.js

```diff
@@ -178,7 +186,16 @@ function addToChangelog(versionText, changelog, lastline) {
    * PACKAGE VERSIONS
    */
   for (const name of Object.keys(pkgJsonPaths)) {
-    writeVersion(pkgJsonPaths[name], input.version);
+    let inputVersion = input.version;
+
+    if (name === 'container' && process.env.NIGHTLY === 'true') {
```

Comment:
On second thought this works pretty well, but I think we also have to think that the same script **release-cli.js** is used for non-Nightly normal releases. 
So in that case **process.env.NIGHTLY** is not true, and that means 

the following will run 
` writeVersion(pkgJsonPaths[name], inputVersion);`
 
 with 
`writeVersion(pkgJsonPaths["container"], <Whatever Version Core has>);` and maybe end up publishing a Container release with a Core version. 

but Core Version !== Container version. 


So to alleviate this issue we can add on lines 41 and 67 above something like: 
```
// only add container to paths if nightly is present. Otherwise container path not needed
if (process.env.NIGHTLY is enabled) {
      pkgJsonPaths.container = path.resolve(base, 'container', 'public', 'package.json')
      installPaths.container = path.resolve(base, 'container'),
}
```